fission-cli/library/Fission/CLI/Handler/User/Register.hs

```diff
@@ -140,6 +137,6 @@ createAccount maybeUsername maybeEmail = do
 
       CLI.Error.put err $
         errMsg <> " Please try again or contact Fission support at https://fission.codes"
-        raise err
-      -- createAccount maybeUsername maybeEmail
+        -- raise err
 
+      createAccount maybeUsername maybeEmail
```

Comment:
Throwing them back into the beginning of the account recovery loop. Maybe you're trying to avoid that for the GH Action, though?